src/heronarts/lx/parameter/BoundedFunctionalParameter.java

```diff
@@ -0,0 +1,121 @@
+package heronarts.lx.parameter;
```

Comment:
Needs copyright header...

Reply:
Oops, late night.  There will be one more of those in P3LX.